kolibri/content/models.py

```diff
@@ -90,7 +90,8 @@ class ContentNode(MPTTModel):
     """
     id = UUIDField(primary_key=True)
     parent = TreeForeignKey('self', null=True, blank=True, related_name='children', db_index=True)
-    license = models.ForeignKey('License', null=True, blank=True)
+    license_name = models.CharField(max_length=50)
```

Comment:
should there be a list of choices for license names?

Reply:
Let me see if it's in le-utils already.

Reply:
Hrm, it is in le_utils, although thinking through the future design for this, I would probably want to err on the side of being permissive on Kolibri, and assume that Studio is doing the work of specifying these license names properly.

My reason being that while we currently have `Special Permission` as a catch all for anything not covered by the other licenses, it is possible that we could introduce new licenses with descriptions that are not `Special Permissions` - as such, I would still like them to make it into the database, rather than throw an error simply because we don't recognize the license.

In the frontend, we will display translated license names and descriptions for recognized licenses, but otherwise just show the DB value for both.

When this frontend is in place, we can also update the import code, so as not to import the description if it is for one of these recognized licenses.

Reply:
In this way, I think it is different to something like `kind`, where we have a real issue handling something that is of a content kind we don't recognize.